packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1424,7 +1425,7 @@ export class ContainerRuntime
 
 		blobManagerLoadInfo: IBlobManagerLoadInfo,
 		private readonly _storage: IDocumentStorageService,
-		private readonly createIdCompressor: () => Promise<IIdCompressor & IIdCompressorCore>,
+		private readonly createIdCompressorFn: () => Promise<IIdCompressor & IIdCompressorCore>,
```

Comment:
If you are removing dynamic import, then I'd assume that all of the functions like that should become synchronous, as there is no reason for them to be async (based on my memory).

Reply:
Made them sync